src/library_dylink.js

```diff
@@ -17,17 +17,7 @@ var LibraryDylink = {
         value = value.value;
       }
       if (typeof value === 'number') {
-        // relocate it - modules export the absolute value, they can't relocate before they export
-#if EMULATE_FUNCTION_POINTER_CASTS
-        // it may be a function pointer
-        if (e.substr(0, 3) == 'fp$' && typeof exports[e.substr(3)] === 'function') {
-          value += tableBase;
-        } else {
-#endif
-          value += memoryBase;
-#if EMULATE_FUNCTION_POINTER_CASTS
-        }
-#endif
+        value += memoryBase;
```

Comment:
maybe keep an assert here for the condition on line 23, that it doesn't happen? we can remove it later, but might help if we forgot something.

Reply:
I did that locally and ran the test suite.. but sure I can land it that way too.

Reply:
Actually I think running the tests locally with this assert is enough.

Reply:
sgtm